src/application/pokegear/phone/phone_generic_headers.c

```diff
@@ -0,0 +1,3243 @@
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+
+    // PHONE_CONTACT_LYRA/1
+    PHONE_GENERIC_TERM,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+
+    // PHONE_CONTACT_LYRA/2
+    PHONE_GENERIC_TERM,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+
+    // PHONE_CONTACT_KURT/1
+    PHONE_GENERIC_TERM,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+
+    // PHONE_CONTACT_KURT/2
+    PHONE_GENERIC_TERM,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+
+    // PHONE_CONTACT_DAY_C_MAN/1
+    PHONE_GENERIC_TERM,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+
+    // PHONE_CONTACT_DAY_C_MAN/2
+    PHONE_GENERIC_TERM,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+
+    // PHONE_CONTACT_DAY_C_LADY/1
+    PHONE_GENERIC_TERM,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+
+    // PHONE_CONTACT_DAY_C_LADY/2
+    PHONE_GENERIC_TERM,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+
+    // PHONE_CONTACT_BUENA/1
+    PHONE_GENERIC_TERM,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+
+    // PHONE_CONTACT_BUENA/2
+    PHONE_GENERIC_TERM,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+
+    // PHONE_CONTACT_BILL/1
+    PHONE_GENERIC_TERM,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+
+    // PHONE_CONTACT_BILL/2
+    PHONE_GENERIC_TERM,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+    PHONE_GENERIC_NULL,
+
+    // PHONE_CONTACT_JOEY/1
+    {
+     .type = 7,
```

Comment:
Can these types be constant'd? I can't exactly follow along with the data aside from this maybe being a refight chance or gift chance